source/help/messaging/sending-messages.rst

```diff
@@ -0,0 +1,60 @@
+.. _sending-messages:
+
+Messaging Basics
+================
+
+After a message has been sent, you or others can reply to it and link to it. Depending on which edition of Mattermost you have and how it's configured by the System Admin, you can also edit and delete your own messages.
+
+Message Types
+-------------
+
+Mattermost has two types of messages: posts and replies.
+
+Posts
```

Comment:
Is there a more end user friendly terminology than posts? Perhaps "messages" and "replies?" Or do we even have to make this distinction in the docs? Perhaps providing a screenshot and putting some of this info below the "Replies" heading would be suffient here.